test/e2e/tests/address-book.spec.js

```diff
@@ -57,6 +57,15 @@ describe('Address Book', function () {
 
         await driver.clickElement({ text: 'Next', tag: 'button' });
 
+        await driver.clickElement({ text: 'Edit', tag: 'button' });
+
+        const [gasLimitInput, gasPriceInput] = await driver.findElements(
+          'input[type="number"]',
+        );
+        await gasPriceInput.fill('8');
+        await gasLimitInput.fill('100000');
+        await driver.clickElement({ text: 'Save', tag: 'button' });
+
```

Comment:
Not sure why is this step required for adding contact to address book.

Reply:
This step was not adding to address book, it's sending to an address book entry. The problem with the e2e was that the Confirm button was not enabled because of invalid gas values.
What these steps do is click Edit which opens the EditGasModal, add the correct values for price and limit and then Save, then the Confirm button is enabled and the tx can be sent.